lib/prx_auth/rails/ext/controller.rb

```diff
@@ -34,10 +35,36 @@ def authenticate!
         redirect_to PrxAuth::Rails::Engine.routes.url_helpers.new_sessions_path
       end
 
+      def prx_auth_needs_refresh?(jwt_ttl)
+        request.get? && jwt_ttl < PRX_JWT_REFRESH_TTL
```

Comment:
This is the check for "should we interrupt the request cycle to refresh our JWT?"

Moved it into its own method so Augury can override.  And ensure we're not in the middle of a Turbo `fetch`.  As currently, a non-full-page Turbo request can't successfully follow the redirect (CORS issues).  Eventually, an upstream Turbo bug/feature may allow us to just break out of Turbo entirely and force the refresh.